### PR TITLE
Expose Location Zones metrics

### PR DESCRIPTION
**Pull request by jjngx**
### Proposed changes

This PR makes [location zones metrics](https://github.com/nginxinc/nginx-prometheus-exporter#location-zones) available for a dashboard / prometheus.

### Checklist
Before creating a PR, run through this checklist and mark each as complete.

- [X] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have checked that all unit tests pass after adding my changes
- [X] I have updated necessary documentation
- [X] I have rebased my branch onto main
- [X] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

**Review by ADubhlaoich — APPROVED** [review]
LGTM!

File: docs/content/logging-and-monitoring/prometheus.md [hunk 1]
```diff
@@ -50,6 +50,13 @@ The Ingress Controller exports the following metrics:
   * `controller_ingress_resources_total`. Number of handled Ingress resources. This metric includes the label type, that groups the Ingress resources by their type (regular, [minion or master](/nginx-ingress-controller/configuration/ingress-resources/cross-namespace-configuration)). **Note**: The metric doesn't count minions without a master.
   * `controller_virtualserver_resources_total`. Number of handled VirtualServer resources.
   * `controller_virtualserverroute_resources_total`. Number of handled VirtualServerRoute resources. **Note**: The metric counts only VirtualServerRoutes that have a reference from a VirtualServer.
+  * `locatio_zone` (services) metrics:
```
Comment (vepatel) [inline docs/content/logging-and-monitoring/prometheus.md:53]: typo `location`

Comment (jjngx) [reply docs/content/logging-and-monitoring/prometheus.md]: fixed

File: docs/content/logging-and-monitoring/prometheus.md [hunk 2]
```diff
@@ -50,6 +50,13 @@ The Ingress Controller exports the following metrics:
   * `controller_ingress_resources_total`. Number of handled Ingress resources. This metric includes the label type, that groups the Ingress resources by their type (regular, [minion or master](/nginx-ingress-controller/configuration/ingress-resources/cross-namespace-configuration)). **Note**: The metric doesn't count minions without a master.
   * `controller_virtualserver_resources_total`. Number of handled VirtualServer resources.
   * `controller_virtualserverroute_resources_total`. Number of handled VirtualServerRoute resources. **Note**: The metric counts only VirtualServerRoutes that have a reference from a VirtualServer.
+  * `location_zone` (services) metrics:
```
Comment (vepatel) [inline docs/content/logging-and-monitoring/prometheus.md:53]: can we change `services` to `upstream services`

Comment (jjngx) [reply docs/content/logging-and-monitoring/prometheus.md]: done

**Review by vepatel — APPROVED** [review]

**Pull request merged**